packages/orchestration/src/facade.js

```diff
@@ -42,6 +45,30 @@ const makeLocalChainFacade = localchain => {
           console.log('deposit got', payment);
           return E(account).deposit(payment);
         },
+        async getAddress() {
+          const addressStr = await E(account).getAddress();
+          return {
+            address: addressStr,
+            chainId: 'agoric-3',
```

Comment:
`chainId` will be `agoriclocal` in a3p, right? seems like `chainId` should be passed in. Or a TODO or something.